plugins/catalog-backend/src/service/AuthorizedEntitiesCatalog.ts

```diff
@@ -151,6 +153,35 @@ export class AuthorizedEntitiesCatalog implements EntitiesCatalog {
     };
   }
 
+  async facets(request: EntityFacetsRequest): Promise<EntityFacetsResponse> {
+    const authorizeDecision = (
```

Comment:
Instead of `[0]` you can destructure the array on assignment:
```
const [authorizeDecision] = (...
```
I find this a tiny bit nicer; but you won't get an index OOB error so must check that you received a value, if you care to know.

Reply:
Ah yeah this was copy pasted from the old entities code above - good call though

Reply:
Leaving as a todo